src/rendering/renderers/gl/context/GlContextSystem.ts

```diff
@@ -8,30 +8,59 @@ import type { WebGLExtensions } from './WebGLExtensions';
 
 /**
  * Options for the context system.
- * @ignore
+ * @memberof rendering
+ * @property {WebGL2RenderingContext | null} [context=null] - User-provided WebGL rendering context object.
```

Comment:
Why are these docs duplicated below? Also I didn't think nested properties on interfaces required memberof.

Reply:
I couldn't figure out a better way to make an interface use `@memberof WebGLOptions` and also still be a member of the original interface